src/providers/wfs/oapif/qgsoapifcollection.cpp

```diff
@@ -298,25 +299,74 @@ bool QgsOapifCollection::deserialize( const json &j )
     }
   }
 
+  // Usage storageCrs from Part 2 in priority
+  bool layerCrsSet = false;
+  if ( j.contains( "storageCrs" ) )
+  {
+    const auto crsUrl = j["storageCrs"];
+    if ( crsUrl.is_string() )
+    {
+      QString crsStr = QString::fromStdString( crsUrl.get<std::string>() );
+      QgsCoordinateReferenceSystem crs = QgsCoordinateReferenceSystem::fromOgcWmsCrs( crsStr );
+
+      if ( j.contains( "storageCrsCoordinateEpoch" ) )
+      {
+        const auto storageCrsCoordinateEpoch = j["storageCrsCoordinateEpoch"];
+        if ( storageCrsCoordinateEpoch.is_number() )
+        {
+          crs.setCoordinateEpoch( storageCrsCoordinateEpoch.get<double>() );
+        }
+      }
+
+      layerCrsSet = true;
+      mLayerMetadata.setCrs( crs );
+      mCrsList.append( crs.authid() );
```

Comment:
why are we using authid here? Does that specifications mandate use of crs from known registries?

Reply:
> Does that specifications mandate use of crs from known registries?

yes, OGCAPI Features Part 2 allows non-WGS84 CRS through http URIs, like http://www.opengis.net/def/crs/EPSG/0/2056